### ISSUE TITLE
refactor neural network normalization to work with sparse matrices

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/data-formatter#30]
should be relatively easy...

convert to csc, get a row, work through that row to normalize it as we normally would
we can even consider todense() for that single row. though then we need to think about converting back to sparse, and i'm not sure if that's easy or not, given that we're integrating into an existing sparse array

**Comment by ClimbsRocks** [comment]
using MaxAbsScaler from the (pre-release) 0.17 version of scikit-learn makes this a breeze.
